tmt/steps/discover/__init__.py

```diff
@@ -47,6 +47,12 @@ def save(self, data=None):
             for test in self.tests()])
         self.write('run.yaml', tmt.utils.dict_to_yaml(tests, width=1000000))
 
+    @property
+    def is_in_standalone_mode(self):
+        if self.opt('fmf_id'):
```

Comment:
I believe the `fmf-id` option handling should be covered under the `DiscoverFmf` class. The general `Discover` class does not know (or should not know, it's weird that it was here) anything about `fmf` plugin options. Here it should be something like `any(plugin.is_in_standalone_mode for plugin in self.plugins())`, what do you think?

Reply:
That's not even necessary since the `any` logic is defined in the general `Step` class so it's not necessary to override it in DiscoverStep